open-apparel-registry/open-apparel-registry
# Add the new homepage

## Overview

Adds the new homepage, including:
- Creating the new homepage layout
- Separating the homepage from the /facilities route
- Updating the filter styles
- Hiding the 'Zoom to Search' button on the homepage
- Updating the embedded map src URL to point to /facilities

Connects #2047 

## Demo

<img width="1431" alt="Screen Shot 2022-08-26 at 8 35 55 AM" src="https://user-images.githubusercontent.com/21046714/186922623-8557d397-5fb4-4cad-bd32-bcb8241fe0c3.png">
<img width="712" alt="Screen Shot 2022-08-26 at 9 44 43 AM" src="https://user-images.githubusercontent.com/21046714/186922641-2dedacd4-b68c-48ba-8f0a-2997bd610e6b.png">
<img width="697" alt="Screen Shot 2022-08-26 at 9 45 04 AM" src="https://user-images.githubusercontent.com/21046714/186922651-18ac909c-4f46-431c-9e5e-0651b18bb9d8.png">
<img width="1429" alt="Screen Shot 2022-08-26 at 9 45 29 AM" src="https://user-images.githubusercontent.com/21046714/186922665-3dfd3cd1-7320-4be1-900c-76735664d35b.png">
<img width="1429" alt="Screen Shot 2022-08-26 at 9 45 50 AM" src="https://user-images.githubusercontent.com/21046714/186922674-26aa25b8-2028-4f50-8d3b-ab2ad7d7b236.png">
<img width="1425" alt="Screen Shot 2022-08-26 at 9 46 08 AM" src="https://user-images.githubusercontent.com/21046714/186922687-05e0df1c-5c1f-439b-86e2-8aa6d3ae2246.png">

## Notes

Some items that were intentionally excluded from this task (due to scope):

- The contributor search tab and tab bar
- Newsletter bar
- New footer
- Any updated filter behaviors
- The new 'autosuggest' search functionality
- Header changes / Google translate changes (aside from a header bug-fix)

We considered updating the /facilities page to default to the Facilities tab showing instead of the Search tab, but the search wasn't running well consistently with that change; I believe fully testing that will be better handled in #2049.

## Testing Instructions

* Run `./scripts/server`
* Visit [the homepage](http://localhost:6543/) and compare to [the wireframes](https://www.figma.com/file/FGVrJYElirWCQyR7nBehF0/OAR?node-id=24%3A25)
     * Note particularly that the 'Zoom to Search' control should be hidden. 
     * The filters should be highlighted in purple when selected, and the selected items should be green 'pills'.
     * Also view the page in mobile sizing and confirm mobile view appears as expected. 
* Run searches for the main filters. You should be taken to the /facilities page with the filters set. 
* Return to [the homepage](http://localhost:6543/). Select a contributor, and then open the drawer by clicking 'more filters'.
     * The drawer should contain all of the other filters. 
     * Setting the filters should update the number in the 'more filters' button.
     * Clicking 'reset' in the drawer should only reset the drawer filters. 
     * The drawer should display correctly in mobile view. 
* Click Search. 
     * The contributor previously selected should have their name displayed in the filter, not their id. 
     * Any selected 'more filters' should be selected and work as expected. 
* Toggle the 'Zoom to Search' button and confirm it works as before. 
* Return to [the homepage](http://localhost:6543/) and use 'Draw Custom Area'  search. You should be taken directly to the '/facilities' page. 

## Checklist
